spec/e2e.rb

```diff
@@ -116,5 +122,42 @@ def exec(cmd)
     def output(line)
       puts line
     end
+
+    def register_config(config)
+      config.before :example, :e2e => true do
+        if not E2E.configured
```

Comment:
Nit: recommend use of `unless` rather than `if not`, eg. `unless E2E.configured?`

Nit: if you update `#deployed` to be a predicate (`#deployed?`), recommend updating `#configured` to `#configured?` for consistency